sql/moz-fx-data-shared-prod/stripe_external/customers_changelog_v1/query.sql

```diff
@@ -0,0 +1,105 @@
+WITH customers AS (
+  -- Get customer records added or updated since the last run.
+  SELECT
+    id,
+    _fivetran_synced,
+    address_country,
+    address_postal_code,
+    address_state,
+    created,
+    default_card_id,
+    is_deleted,
+    PARSE_JSON(metadata) AS metadata,
```

Comment:
While the unhashed FxA user IDs in `metadata` are no longer verboten, there's no telling what kinds of data they might put in `metadata` in the future, which is why I'm adding this ETL in the restricted `stripe_external` dataset rather than in the Moz-confidential `subscription_platform_derived` dataset.

I'll also probably move `subscription_platform_derived.stripe_subscriptions_changelog_v1` into the `stripe_external` dataset and switch it back to pulling in all the metadata (the main reason I changed that ETL to parse specific metadata in #4009 was the realization that potentially sensitive metadata fields might be added in the future).